<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="PHQVMW7065" name="com.paf.poeprocess.TC2_ApprovePOE" tests="1" failures="0" timestamp="28 Oct 2016 17:00:20 GMT" time="88.390" errors="1">
  <testcase name="test" time="88.390" classname="com.paf.poeprocess.TC2_ApprovePOE">
    <error type="atu.testng.reports.exceptions.ATUReporterStepFailedException">
      <![CDATA[[ATU Custom Reporter Step Failed Exception]
	at atu.testng.reports.ATUReports.stepFailureHandler(Unknown Source)
	at atu.testng.reports.ATUReports.add(Unknown Source)
	at com.paf.poeprocess.TC2_ApprovePOE.viewdocument(TC2_ApprovePOE.java:874)
	at com.paf.poeprocess.TC2_ApprovePOE.test(TC2_ApprovePOE.java:277)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:393)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
]]>
    </error>
  </testcase> <!-- test -->
</testsuite> <!-- com.paf.poeprocess.TC2_ApprovePOE -->
